lib/features/keyboard/Keyboard.js

```diff
@@ -1,14 +1,40 @@
+import {
+  assign,
+  forEach,
+  isArray,
+  isFunction,
+  map
+} from 'min-dash';
 import {
   event as domEvent,
   matches as domMatches
 } from 'min-dom';
 
+import {
+  hasModifier,
+  isCmd,
+  isShift
+} from './helpers';
+
+var DEFAULT_SPEED = 50;
 
 /**
  * A keyboard abstraction that may be activated and
  * deactivated by users at will, consuming key events
  * and triggering diagram actions.
  *
+ * For pressed keys, keyboard fires an event according to following scheme:
+ * `keyboard.keydown.[event.key || event.keyCode]`
```

Comment:
When will this be called with the keyCode? I.e. how to separate the `1` key from the `1` keyCode?

Reply:
Removed the fallback.